StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1004UnitTests.cs

```diff
@@ -213,5 +228,33 @@ private void Method1(int x, int y)
 
             await VerifyCSharpFixAsync(testCode, expected, fixedCode, CancellationToken.None).ConfigureAwait(false);
         }
+
+        [Theory]
+        [MemberData(nameof(ParameterModifiers))]
+        [WorkItem(3817, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3817")]
+        public async Task TestParameterModifierFirstOnLineAsync(string keyword)
+        {
+            string testCode = $@"
+/// <summary>
+/// Description of some remarks that refer to a method: <see cref=""SomeMethod(int, int,
+/// {keyword} string)""/>.
```

Comment:
Do we also need to check this wrapping?

```
/// <see cref="SomeMethod(int, int, ref
/// readonly string)"/>
```

Reply:
To me it seems similar to the current report. I think most users would not wrap a comment in the middle of a `cref` attribute, but automatic word wrapping tools can do it.

Reply:
Ok. Added.